modules/miscutil/lib/__init__.py

```diff
@@ -24,3 +24,7 @@
 import sys
 reload(sys)
 sys.setdefaultencoding('utf8')
+
+## Because we use getLogger calls to do logging, handlers aren't initialised
+## unless we explicitly call/import this code somewhere.
+import elasticsearch_logging
```

Comment:
As discussed IRL this should be handled more carefully otherwise `make install` will fail. 
```console
/usr/bin/python ./testimport.py /opt/invenio
   ******************************************************
    ** IMPORT ERROR: No module named config
    ******************************************************
    ** Perhaps you need to create symbolic link(s)      **
    ** from your system-wide Python site-packages       **
    ** directory(ies) to your Invenio installation  **
    ** directory?                                       **
    **                                                  **
    ** If yes, then please create symlink(s) via:       **
    **
    **    $ sudo ln -s /opt/invenio/lib/python/invenio /usr/lib64/python2.6/site-packages/invenio
    **    $ sudo ln -s /opt/invenio/lib/python/invenio /usr/lib/python2.6/site-packages/invenio
    **
    ** and continue with the 'make install' afterward.  **
    **                                                  **
    ** If not, then please inspect the above error      **
    ** message and fix the problem before continuing.   **
    ******************************************************
```